doc/stability_release.md

```diff
@@ -0,0 +1,28 @@
+# Semantic Versioning
+
+WAMR has adopted [semantic versioning](https://semver.org/) to replace the former *date versioning system*.  The new version string consists of three parts:
+
+- *major*: Any change that is not compatible with previous versions, affecting either the ABI or APIs, will result in an increase in the major version number. APIs include: wasm_export.h, wasm_c_api.h, sections in AOT files, among others.
+- *minor*: This number increases with the addition of new features. This encompasses not only MVP (Minimum Viable Product) or POST-MVP features but also WebAssembly System Interface (WASI) features and WAMR-specific features.
+- *patch*: This number is incremented for patches.
+
+## Legacy releases
+
+All previous versions (tags) will retain their current status. There will be no changes to existing release names and links.
+
+# Release Process
+
+WAMR has been deployed across various devices. A frequent release cycle would strain customers' testing resources and add extra deployment work. Two factors can trigger a new WAMR release:
```

Comment:
> frequent release cycle would strain customers' testing resources

Not sure if I agree with that; customers can still choose to stick to a particular version or keep moving to the latest one. In fact, for some of the usecases it more frequent releases are better as it helps narrowing down any potential regressions which can only be seen when running WAMR at scale.

> and add extra deployment work

Agree with that, we should probably discuss automating the process within TSC as much as we can to streamline the releases.

Reply:
Are you proposing that we should streamline the release process (reduce manual work compared to the current method) and issue releases on a regular schedule, such as monthly?

Reply:
yes, I think that'd be great to have predictable release schedule. Not sure when do we have the next TSC meeting, but let's add it to the agenda.

Reply:
February 3rd falls within the Chinese public holidays, so I plan to cancel the February community meeting. 

Perhaps we should open a new issue to discuss a predictable release schedule. We're completely on board with the idea. The only issue is carrying out the release tests, which are currently done manually. If you're interested, I think I could share the details of the release test cases.